test/SparqlGenerator-test.js

```diff
@@ -38,7 +39,8 @@ describe('A SPARQL generator', function () {
 
     it('should remove unused prefixes from "' + query + '"', function () {
       var parsedQuery = parseJSON(fs.readFileSync(parsedQueryFile, 'utf8'));
-      var expectedQuery = fs.readFileSync(generatedQueryFile, 'utf8');
+      var expectedQuery = fs.readFileSync(generatedQueryFile, 'utf8')
+        .split(os.EOL).join('\n');
```

Comment:
I think we can just `.replace(/[\n\r]+/g, '\n')`

Reply:
@joachimvh You running into this problem as well?

Reply:
No but I do have my git setup to use LF instead of CRLF line endings. I did just try manually storing some of the files with CRLF line endings now and the tests still work, so not sure what the problem is.

Reply:
@RubenVerborgh `.replace(/[\n\r]+/g, '\n')` will remove all empty lines. I use default Git setup for multiplatform projects on Windows, so it converts `\n` to `\r\n` on checkout; see `core.autocrlf` section here: https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration

Reply:
Right, and we have empty lines after prefixes.

How about `.replace(/\n\r/g, '\n')` then? Feel free to amend and I'll merge.